include/sdf/sdf_config.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef SDF_SDF_CONFIG_H_
+#define SDF_SDF_CONFIG_H_
+
+// TODO(anyone): deprecate this header?
```

Comment:
Same comments about changing the name in the `TODO` note apply here

Reply:
removed the comment in https://github.com/ignitionrobotics/sdformat/pull/780/commits/a13f9e5e65f8f941619eb9dd41324bad89981875